packages/labs/context/src/test/context-provider_test.ts

```diff
@@ -82,3 +82,57 @@ suite('@contextRequest', () => {
     assert.strictEqual(consumer.value, 500);
   });
 });
+
+suite('@contextProvided: multiple instances', () => {
+  let consumers: ContextConsumerElement[];
+  let providers: ContextProviderElement[];
+  let container: HTMLElement;
+  const count = 3;
+  setup(async () => {
+    container = document.createElement('div');
+    container.innerHTML = new Array(count)
+      .fill(0)
+      .map(
+        (_v, i) => `
+        <context-provider value="${1000 + i}">
+            <context-consumer></context-consumer>
+        </context-provider>`
+      )
+      .join('/n');
+    document.body.appendChild(container);
+
+    providers = Array.from(
```

Comment:
you don't need the Array.from if you spread below. `querySelectorAll` is generic too, so you can do `querySelectorAll<ContextProviderElement>('context-provider')`

Reply:
Done.